Source/rotation/Rotation.H

```diff
@@ -63,12 +93,21 @@ rotational_acceleration(GpuArray<Real, 3>& r, GpuArray<Real, 3>& v,
 
   bool c2 = (castro::rotation_include_coriolis == 1 && coriolis) ? true : false;
 
+  auto r_vec = r;
+
   GpuArray<Real, 3> omega_cross_v;
   cross_product(omega, v, omega_cross_v);
 
   if (c1) {
+      // For Spherical coordinate, the physical position vector is just r rhat,
+      // but the input r will be in the format of (r,theta,0). So change it to (r,0,0)
```

Comment:
why can we do this?

Reply:
in spherical coord, the position vector is just $r \hat{r}$, so theta should be zeroed out when we do the cross products I think.

Reply:
Oh, this is because it is the position vector from the origin?